### PR TITLE
Amj/369 line scan

### PR DESCRIPTION
**Pull request by jayich**
Review: @aransfor 

Seems to work.

### CODE REVIEW

File: scripts/experiments/dds_linescan/dds_linescan.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    @classmethod
+    def all_required_parameters(cls):
+        return cls.exp_parameters
+
+    def initialize(self, cxn, context, ident):
+        self.p = self.parameters
+        self.ident = ident
+
+        self.cxn = labrad.connect(name='Wavemeter Line Scan')
```
Comment (aransfor) [inline scripts/experiments/dds_linescan/dds_linescan.py]: This needs a new name

**Comment by aransfor** [comment]
LGTM other than the note, is there a scan parameter for the frequencies?

File: scripts/experiments/dds_linescan/dds_linescan.py (hunk 1 above)
Comment (jayich) [inline scripts/experiments/dds_linescan/dds_linescan.py]: Thanks for the catch.

**Comment by jayich** [comment]
There is a scan parameter for the 369 frequency.

**Pull request merged**